tests/check-legal-aid/test/browserstack-integration.json

```diff
@@ -0,0 +1,104 @@
+{
+  "src_folders" : "tests/nightwatch/specs",
+  "output_folder" : "tests/reports/nightwatch",
+  "custom_assertions_path" : "tests/nightwatch/custom_assertions",
+  "custom_commands_path" : "tests/nightwatch/commands",
+
+  "selenium" : {
+    "start_process" : false,
+    "host" : "hub.browserstack.com",
+    "port" : 80
+  },
+
+  "test_settings" : {
+    "default" : {
+      "launch_url" : "http://public-integration.cla.dsd.io",
```

Comment:
Could we remove this configuration until it's necessary, please?

Reply:
:thumbsup: